### ISSUE TITLE
Slur filter hostile to marginalized users (and other concerns)

### ISSUE THREAD

**Issue by hdkdhlskw** [LemmyNet/lemmy#1466]
Disclaimer: I am a multiply marginalized person on the radical left.

I see various issues with the slur filter. 

The biggest one, I feel, is that many, many people in marginalized communities have reclaimed slurs. I'd go as far as to say that some (myself included) strongly identify with reclaimed slurs. The word "queer" is a very common example. Will those who identify with it not be allowed to express themselves fully here? The ban on slurs actually makes me feel *far* less welcome here as a marginalized person as a part of my identity that I am proud of, embrace, and find power in is banned. Most of my friends with various marginalizations have reclaimed slurs as well and would not feel welcome in this space. The reclamation of slurs can be an essential tool for marginalized people. Who are non-marginalized people to decide which slurs marginalized people are allowed to reclaim? I encourage you to read more about this, because it is incredibly important. 

Additionally, the code used to filter slurs is flawed. Does it handle if users use alternate Unicode characters to write slurs? Replacing "O"s with "0"s? Slur filters have been implemented time and time again and the result is always the same: users get more creative in their use of slurs or even invent new ones. There are so many variations of slurs, and language is far too complex for this to be enforced with a simple regex. It's also critical to consider different languages here. If Lemmy centers English in its slur filtering, it will inadvertently censor non-English words that *are not* slurs as well as not censoring non-English words that *are*. Not to mention -- centering English is incredibly problematic. 

Finally, the code is easily removed, and I speculate that if anything, it will lead to a fork of Lemmy by the alt-right even sooner that will gain significant traction. At the very least, marginalized users such as myself who simply wish to reclaim slurs will have to go through the labour of modifying the code and hosting our own instances. 

**tldr: as a multiply-marginalized person with experience developing and running community platforms, this is a huge mistake, and will end up alienating many of those that you wish to protect.**

Please reconsider this change as it is far more nuanced than it appears on the surface. Thank you.

**Comment by dessalines** [comment]
We already have issues for this.
